examples/hcl/linux/variables.common.pkr.hcl

```diff
@@ -20,4 +20,10 @@ locals {
   // value. This validates that the http directory exists even before starting
   // any builder/provisioner.
   http_directory = dirname(convert(fileset(".", "etc/http/*"), list(string))[0])
+  http_directory_content = {
+    "alpine-answers"           = file("${local.http_directory}/alpine-answers"),
```

Comment:
It would be way better to just use `http_directory` here, but that's just for testing purposes